wgpu-types/src/lib.rs

```diff
@@ -1786,7 +1786,8 @@ impl<L> TextureViewDescriptor<L> {
     }
 
     pub fn level_count(&mut self, level_count: u32) -> &mut Self {
-        self.level_count = Some(level_count);
+        self.level_count
```

Comment:
why are we replacing instead of just assigning to it?

Reply:
No particular reason. Is there a downside to it?

Reply:
assignment is simpler. In the lack of other criteria, choose the simplest one